prosessering-core/src/main/kotlin/no/nav/familie/prosessering/config/ProsesseringConfig.kt

```diff
@@ -19,8 +27,28 @@ class ProsesseringConfig(@Value("\${prosessering.queue.capacity:20}") private va
         executor.maxPoolSize = poolSize
         executor.threadNamePrefix = "TaskWorker-"
         executor.setWaitForTasksToCompleteOnShutdown(true)
+        executor.setAwaitTerminationSeconds(20)
         executor.setQueueCapacity(køstørrelse)
         executor.initialize()
         return executor
     }
+
+    @Bean
+    fun threadPoolTaskScheduler(): ThreadPoolTaskScheduler {
+        val executor = ThreadPoolTaskScheduler()
+        executor.threadNamePrefix = "TaskScheduler-"
+        executor.setWaitForTasksToCompleteOnShutdown(true)
+        executor.setAwaitTerminationSeconds(20)
+        executor.setErrorHandler { e ->
```

Comment:
Virket ikke som att det fantes errorHandler for `ThreadPoolTaskExecutor` (i den andre bønan).